### PR TITLE
Add-validation

### PR DESCRIPTION
**Pull request by nobu0605**
I added isEmpty and validateAddress function.

### CODE REVIEW

**Comment by TakanobuSuzuki0605** [comment]
Please add screen shot.

File: src/components/Map.tsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ export default class Map extends React.Component<Props, State> {
       companyName: companyName,
       isShowInfoWindow: true,
       lat: lat - adjustLatitude,
-      lng: lng,
+      lng: lng - 3,
```
Comment (TakanobuSuzuki0605) [inline src/components/Map.tsx:89]: Please change this number to constant.

File: src/pages/ShopList.tsx [hunk 2]
```diff
@@ -21,6 +22,15 @@ export default class ShopList extends React.Component<Props, State> {
 
   filterList(e: React.ChangeEvent<HTMLInputElement>) {
     e.preventDefault();
+    // console.log(e.target.value);
```
Comment (TakanobuSuzuki0605) [inline src/pages/ShopList.tsx:25]: Please remove this comment out.

File: src/utils/validation.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export function isEmpty(value) {
+  return typeof value === "undefined" || value === null || value.length === 0;
+}
+
+export function validateAddress(address, characterlimit) {
```
Comment (TakanobuSuzuki0605) [inline src/utils/validation.js:5]: Could you change this argument name to the camel case?

File: src/components/Map.tsx (hunk 1 above)
Comment (nobu0605) [reply src/components/Map.tsx:89]: Good catch!

File: src/pages/ShopList.tsx (hunk 2 above)
Comment (nobu0605) [reply src/pages/ShopList.tsx:25]: Got it!

**Pull request merged**